bundles/org.eclipse.cdt.lsp.clangd/src/org/eclipse/cdt/lsp/internal/clangd/editor/ClangdConfigurationPage.java

```diff
@@ -221,8 +219,9 @@ public boolean performOk() {
 		return true;
 	}
 
-	private boolean isLspEditorOpen() {
-		return LspUtils.getEditors().stream().anyMatch(e -> LspPlugin.LSP_C_EDITOR_ID.equals(e.getId()));
+	private boolean isLsActive() {
+		return LanguageServiceAccessor.getStartedWrappers(null, null, true).stream()
+				.filter(w -> "org.eclipse.cdt.lsp.server".equals(w.serverDefinition.id)).findAny().isPresent(); //$NON-NLS-1$
```

Comment:
(minor) The ID string risks proliferating ([its here already](https://github.com/eclipse-cdt/cdt-lsp/blob/b583a130fe9d86068229785c3fc78c4475a58cec/bundles/org.eclipse.cdt.lsp.clangd/src/org/eclipse/cdt/lsp/internal/clangd/editor/ClangdConfigurationPage.java#L236)) have a shared constant for it makes sense to me.
It may even be worth adding the whole `LanguageServiceAccessor.getStartedWrappers(null, null, true).stream().filter(w -> "org.eclipse.cdt.lsp.server".equals(w.serverDefinition.id))` to `LspUtils`

Reply:
You're right. I'll move it to `LspUtils`